cms/templates/cms/home_page.html

```diff
@@ -215,19 +210,15 @@ <h2 class="quote-text">"Inverted admissions has the potential to disrupt traditi
         <div class="modal-body">
           <h2>MicroMasters is a new credential</h2>
           <p>
-          MIT has created a new credential for online learners: the “MicroMasters,” which will be granted by MITx
-          (MIT’s online learning initiative) to students who do exceptionally well in a given set of graduate-level
-          online courses and do well in a subsequent exam. The credential will first be available to students who register for the SCM program’s online courses.
+          MIT has created a new credential for online learners: the “MicroMasters,” which will be granted by MITx (MIT’s online learning initiative) to students who do exceptionally well in a given set of graduate-level online courses, and in subsequent exam(s).
           </p>
           <h3>Join the MITx MicroMasters community</h3>
           <p>
-          MicroMasters aren't just about courseware. Part of the the MIT experience is interacting with other exceptional
-          students like yourself. Fill out your profile on this site and build professional relationships with like-minded learners.
+          MicroMasters isn't just about courseware. Part of the the MIT experience is interacting with other exceptional students like yourself. Fill out your profile and build professional relationships with like-minded learners.
           </p>
           <h3>MicroMasters courses are delivered through edX.org</h3>
           <p>
-          Use your edX.org account to enroll in a MicroMasters program and share your grades and progress with MIT.
-          When you complete all the requirements and earn a MicroMasters credential, you can use it to apply for a Master’s degree at MIT.
+          Use your edX.org account to enroll in a MicroMasters program and share your grades and progress with MIT. When you complete all the requirements and earn a MicroMasters credential, you can use it to apply for a Master’s degree at MIT.
```

Comment:
I don't think so, unless it's a link that behaves the same as the sign-up button.